pkg/datapath/linux/probes/managed_neighbors.go

```diff
@@ -83,9 +83,6 @@ func haveManagedNeighbors() (outer error) {
 		return fmt.Errorf("failed to add dummy veth: %w", err)
 	}
 
-	if err := netlink.LinkAdd(veth); err != nil {
-		return fmt.Errorf("failed to add dummy veth: %w", err)
-	}
```

Comment:
Since this was for testing can we drop the change from both commits? I imagine this could cause problems during a bisect

Reply:
@thorn3r Thank you for the review! I'll axe it